### DISCUSSION TITLE
What is meant by detailed traffic statistics when using the UI_TRAFFIC_STATS option

### DISCUSSION THREAD

**Discussion by djparus** [wg-easy/wg-easy#1430] [Q&A]
I tried adding `--env UI_TRAFFIC_STATS=true \`
 to my docker container, but I still couldn't see detailed statistics in the UI interface. Everything remains as it was before adding `UI_TRAFFIC_STATS`

**Comment by 510846** [comment]
the same question

**Comment by kaaax0815** [comment]
Not much is changed actually. The UI slightly changes a bit tho. The screenshot in the README is with it disabled. The image in here https://github.com/wg-easy/wg-easy/issues/1111 shows it enabled.

TLDR: You also see the Total used Bandwidth (not only the current)
